mcd-sel-fj/mcd-sel-fj.py

```diff
@@ -92,10 +92,14 @@
     for submenu in submenus:
 
         # Category Names should only contain alphabetical characters
+        category_id = submenu.get_attribute("id")
         category = " ".join(
             re.findall(
                 r"[a-zA-Z]+",
-                ((submenu.find_element(By.CSS_SELECTOR, "h2")).text)
+                ((browser.find_element(
+                    By.CSS_SELECTOR,
+                    f'nav.scrollspy-menu a[data-slug="{category_id}"]'
+                )).text)
```

Comment:
Text in anchor elements are rendered in proper Pascal Case on the webpage.
Hence, we are switching selectors to get text from these elements instead.

The `id` attribute of each html section element conveniently corresponds to the `data-slug` attribute of each anchor element.